addons/mail/static/src/components/attachment_viewer/attachment_viewer.js

```diff
@@ -582,7 +582,9 @@ class AttachmentViewer extends Component {
 }
 
 Object.assign(AttachmentViewer, {
-    props: {},
+    props: {
+        localId: String, // needed by dialogManager
+    },
```

Comment:
You could just have put this and ignore using this prop.

Reply:
Yes, but he's going to create a record nevertheless which is the one shown, so no attachment will be available if we use the one in messaging in the template